src/builder.js

```diff
@@ -98,9 +98,11 @@ builder.schedule = function(repo, gitBranch, uuid, dockerOptions) {
       project.homepage        = repo;
       project['github-token'] = githubToken;
       project.registry        = project.registry || '127.0.0.1:5000';
+      dockerOptions.dockerfile = project.dockerfile;
 
       console.log('project ' + name + ': ', utils.obfuscate(project));
       if (!!project.build) {
+        dockerOptions.finalDockerfile = dockerOptions.dockerfile;
```

Comment:
Yes 

Sent from my iPhone

> On Apr 4, 2016, at 9:38 AM, Alessandro Nadalin <notifications@github.com> wrote:
> 
> In src/builder.js:
> 
> >  
> >        console.log('project ' + name + ': ', utils.obfuscate(project));
> >        if (!!project.build) {
> > +        dockerOptions.finalDockerfile = dockerOptions.dockerfile;
> do we need both the dockerfile and finalDockerfile?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>